library/RevAPI/Exception/RequestException.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace RevAPI\Exception;
+use Guzzle\Http\Exception\BadResponseException;
+
+class RequestException extends \Exception {
```

Comment:
Should this extend `\RevAPI\Exception`? Not quite sure what the inheritance chain should look like.